### PR TITLE
Default values.yaml for ArgoCD helm on spoke

### PR DESCRIPTION
**Pull request by iamahgoub**
*Issue #, if available:*

*Description of changes:*

Adding default values.yaml for ArgoCD helm release run on spoke clusters. For now, it only changes the type of the service to LoadBalancer to expose the UI so attendees can easily explore it during the workshop.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: gitops/fleet/charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+server:
+  service:
+    type: LoadBalancer
```
Comment (csantanapr) [inline gitops/fleet/charts/argo-cd/values.yaml:3]:
```suggestion
    type: LoadBalancer

```

Comment (csantanapr) [inline gitops/fleet/charts/argo-cd/values.yaml:3]: Did you tested the terraform spoke destroy.sh deletes the loadbalancer here? https://github.com/aws-samples/fleet-management-on-amazon-eks-workshop/blob/riv24/terraform/spokes/destroy.sh#L29

Comment (iamahgoub) [reply gitops/fleet/charts/argo-cd/values.yaml:3]: I found the ns for argocd on spoke incorrect in destroy.sh; I corrected that in the last commit.

Comment (iamahgoub) [reply gitops/fleet/charts/argo-cd/values.yaml:3]: I tested the destroy.sh script after fixing the ns, and it worked.

**Pull request merged**